### PR TITLE
Added Testimonial Wall page (Testimonial backend)

### PR DESCRIPTION
**Pull request by theBeginner86**
## What is the change?
In this PR, I have added the testimonial wall page which works using simple get requests in testimonial.router.js and then renders the ejs files.

## Related issue?
close: #787 

## How was it tested?
tested locally

## Checklist:
Before you create this PR, confirm all the requirements listed below by checking the checkboxes `[x]`:

- [x] Have you followed the [Contribution Guidelines](https://github.com/ALPHAVIO/BlogSite/blob/master/CONTRIBUTING.md) while contributing.
- [x] Have you checked there aren't other open [Pull Requests](https://github.com/ALPHAVIO/BlogSite/pulls) for the same update/change?
- [] Did you lint your code before making the Pull Request?
- [] Have you made corresponding changes to the documentation?
- [x] Your submission doesn't break any existing feature.
- [x] Have you tested the code before submission?

## Screenshots or Video:
Add a screenshot or demo video if appropriate.
Screenshot 2021-05-01 at 2 37 08 AM
Screenshot 2021-05-01 at 11 18 00 AM

### CODE REVIEW

**Comment by theBeginner86** [comment]
I had to force push twice because I had put some comments and console.log in the files and I realised it later. But I believe that there aren't any conflicts.

File: views/postitems/testimonial-wall.ejs [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<%- include("../partials/header") -%>
+
+<div class="container testimonial-wall">
+    <h1>Testimonials</h1>
```
Comment (Pranav016) [inline views/postitems/testimonial-wall.ejs:4]: Can you make a separate folder in `views` for all the testimonial related files ?
I know the testimonial-write file was already present in `postitems` but I feel that we'll be having more files for testimonials then we should have separate folder as well to ensure future scalability.

Also transfer the testimonial write file to the new testimonial folder you make and update relative paths.

File: public/css/styles.css [hunk 2]
```diff
@@ -931,6 +931,19 @@ input[name='perPage'] {
 	margin-top: -0.313rem;
 }
 
+.testimonial-wall {
+	padding: 50px;
+}
+
+.testimonial-data {
+	padding-top: 30px;
+}
```
Comment (Pranav016) [inline public/css/styles.css:940]: don't hard code values in `px`, instead use relative units to ensure responsiveness

**Review by Pranav016 — CHANGES_REQUESTED** [review]
A couple changes otherwise PR looks good

**Comment by Pranav016** [comment]
/gssoc/

**Comment by Pranav016** [comment]
/level3/

**Comment by theBeginner86** [comment]
ok I will try to do all those things.

**Review by Pranav016 — APPROVED** [review]

**Pull request merged**